From 0b56eb62d26348d20bd7ea89386769710826ec83 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Fri, 20 Apr 2012 18:27:50 +0800
Subject: [PATCH 1355/2463] ENGR00179636-03 - i.MX6 : Enable ethernet NAPI
 method in default.

- Enable ethernet NAPI method in default, which can reduce cpu
  loading and RX FIFO overruns in busy system.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 85bdfca..b0a4445 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -957,6 +957,7 @@ CONFIG_SMSC911X=y
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_FEC=y
+CONFIG_FEC_NAPI=y
 # CONFIG_FEC_1588 is not set
 # CONFIG_FTMAC100 is not set
 # CONFIG_NETDEV_1000 is not set
-- 
1.7.7.4

